subprojects/plugins/src/main/resources/org/gradle/api/internal/plugins/unixStartScript.txt

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
❌ We are making an effort in using strict Bourne Shell and avoid any Bashims. Do you think you can fix the issue using only what is available on POSIX?